docs/OV_Runtime_UG/deployment/conditional_compilation_deployment.md

```diff
@@ -0,0 +1,289 @@
+
+Below is an example:
+
+Stage 1: Selective build analyzed stage
+++++++++++++++++++++++++++++++++++++++++
+
+    Build OpenVINO with conditional compilation enabled:
+
+    .. code-block:: sh
+
+        call C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvar64.bat
+        set OPENVINO_HOME=D:\work_path\openvino
+        cd %OPENVINO_HOME%
+        md build_cc
+        cd build_cc
+        cmake -G Ninja -Wno-dev -DCMAKE_BUILD_TYPE=Debug -DENABLE_CPPLINT=OFF -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF -DENABLE_FASTER_BUILD=ON -DENABLE_SANITIZER=OFF -DTHREADING=TBB -DBUILD_SHARED_LIBS=OFF -DENABLE_PROFILING_ITT=ON -DSELECTIVE_BUILD=COLLECT -DENABLE_INTEL_GPU=OFF  -DENABLE_INTEL_GNA=OFF -DENABLE_MULTI=OFF -DENABLE_AUTO=OFF -DENABLE_AUTO_BATCH=OFF -DENABLE_HETERO=OFF -DENABLE_TEMPLATE=OFF -DENABLE_OV_ONNX_FRONTEND=OFF -DENABLE_OV_PADDLE_FRONTEND=OFF -DENABLE_OV_PYTORCH_FRONTEND=OFF -DENABLE_OV_TF_FRONTEND=OFF -DCMAKE_INSTALL_PREFIX=install ..
+
+        cmake --build . --config Debug
+
+
+    Collect statistics data
+
+    .. code-block:: sh
+
+        cd %OPENVINO_HOME%\build_cc
+        cmake --build . --config Debug --target sea_itt_lib
+        cd %OPENVINO_HOME%
+        set PATH=%PATH%;%OPENVINO_HOME%\\temp\tbb\bin
+        mkdir cc_data
+        cd %OPENVINO_HOME%\cc_data
+        python3 ..\thirdparty\itt_collector\runtool\sea_runtool.py --bindir ..\bin\intel64\Debug -o %OPENVINO_HOME%\cc_data\data ! ..\bin\intel64\Debug\benchmark_app.exe -niter 1 -nireq 1 -m <your_model.xml>
+
+.. note::
+
+    This stage is for profiling data, so build with Debug or Release will be up to you.
+
+
+Stage 2: build resulting OpenVINO package
++++++++++++++++++++++++++++++++++++++++++
+
+Generate final optimal binaries size of OpenVINO package
+
+    .. code-block:: sh
+
+        cd %OPENVINO_HOME%
+        md build
+        cd build
+
+        cmake -G "Visual Studio 16 2019" -A x64 -DENABLE_CPPLINT=OFF -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF -DCMAKE_BUILD_TYPE=Release -DENABLE_FASTER_BUILD=ON -DENABLE_PROFILING_ITT=OFF -DSELECTIVE_BUILD=ON -DENABLE_INTEL_GPU=OFF -DENABLE_INTEL_GNA=OFF -DENABLE_MULTI=OFF -DENABLE_AUTO=OFF -DENABLE_AUTO_BATCH=OFF -DENABLE_HETERO=OFF -DENABLE_TEMPLATE=OFF -DENABLE_OV_ONNX_FRONTEND=OFF -DENABLE_OV_PADDLE_FRONTEND=OFF -DENABLE_OV_PYTORCH_FRONTEND=OFF -DENABLE_OV_TF_FRONTEND=OFF -DSELECTIVE_BUILD_STAT=%OPENVINO_HOME%\cc_data\*.csv -DBUILD_SHARED_LIBS=OFF -DENABLE_LTO=ON -DENABLE_ONEDNN_FOR_GPU=OFF -DENABLE_GAPI_PREPROCESSING=OFF -DENABLE_OV_TF_LITE_FRONTEND=OFF -DENABLE_CLDNN=OFF -DENABLE_PROFILING_FIRST_INFERENCE=OFF -DENABLE_INTEL_MYRIAD_COMMON=OFF -DENABLE_INTEL_MYRIAD=OFF ..
+
+        cmake --build . --config Release
+
+
+.. tip::
+
+    ``-DSELECTIVE_BUILD=ON`` and ``-DSELECTIVE_BUILD_STAT=%OPENVINO_HOME%\cc_data\*.csv`` is must, and ``-DBUILD_SHARED_LIBS=OFF`` is recommended.
```

Comment:
Could we have an indication here of being able to give multiple .csv files to represent multiple model stats?

Reply:
`*.csv` means to apply all csv files.